cmd/install.go

```diff
@@ -16,7 +16,8 @@ var installOptions install.Options
 var installCmd = &cobra.Command{
 	Use:   "install",
 	Short: "Install KubeArmor in a Kubernetes Cluster",
-	Long:  `Install KubeArmor in a Kubernetes Clusters`,
+	Long: `Safeguard your Kubernetes cluster effortlessly with KubeArmor CLI installation. This streamlined process equips your 
```

Comment:
The goal of long description is to explain about how KubeArmor autodetects environment and installs things accordingly.

As well explain different ways to use it
karmor install --save
karmor install --env=<specify env>
or passing namespace.

Let's keep the description as functional as possible.